### ISSUE TITLE
A Unified Architecture for Natural Language Processing: Deep Neural Networks with Multitask Learning

### ISSUE THREAD

**Issue by Hironsan** [arXivTimes/arXivTimes#117]
## 一言でいうと
NLPの様々なタスク(NER, POS, チャンキング, 言語モデル, SRL)を同時に学習させることで汎化性能を向上させようという話。具体的には同時に学習するためのニューラルネットワークの枠組みを提案している。結果として、タスク単体で学習させるより複数のタスクを同時に学習させた方が良い結果となった。

### 論文リンク
http://citeseerx.ist.psu.edu/viewdoc/download?doi=10.1.1.149.8551&rep=rep1&type=pdf

### 著者/所属機関
Ronan Collobert, Jason Weston

**Comment by icoxfog417** [comment]
#47 に似てるね。こちらは2008年なのか。

**Comment by Hironsan** [comment]
そうですね。やっていることは似ていますね。なぜか [#47](https://github.com/arXivTimes/arXivTimes/issues/47) の参考文献には載っていませんでしたけど。

**Comment by icoxfog417** [comment]
今後はさらに上位のタスクまで応用が進んでくるかもね。Fine-Tuning的に下層のタスクで学習済みのモデルが使えるとGOODだが。

**Comment by Hironsan** [comment]
この論文では獲得したword embeddingを使い回していますね。

**Comment by icoxfog417** [comment]
こちらのまとめでも参照されていた（橋本さんの#47も）。

[An Overview of Multi-Task Learning in Deep Neural Networks](http://sebastianruder.com/multi-task/index.html#focusingattention)

**Comment by icoxfog417** [comment]
こちらのまとめでも参照されていた（橋本さんの #47  も）。

[An Overview of Multi-Task Learning in Deep Neural Networks](http://sebastianruder.com/multi-task/index.html#focusingattention)
